set-env-mysql.sh

```diff
@@ -0,0 +1,20 @@
+if [ -z "$DOCKER_HOST_IP" ] ; then
```

Comment:
Move the duplicate code in set-env-*.sh into a set-env.sh that both of them `. ./set-env.sh`

Reply:
no problem